types/glob-watcher/index.d.ts

```diff
@@ -0,0 +1,167 @@
+         * watch files in other non-standard situations. Setting to `true` explicitly on OS X overrides
+         * the `useFsEvents` default.
+         */
+        usePolling?: boolean;
+
+        /**
+         * Whether to use the `fsevents` watching interface if available. When set to `true` explicitly
+         * and `fsevents` is available this supercedes the `usePolling` setting. When set to `false` on
+         * OS X, `usePolling: true` becomes the default.
+         */
+        useFsEvents?: boolean;
+
+        /**
+         * If relying upon the [`fs.Stats`](http://nodejs.org/api/fs.html#fs_class_fs_stats) object that
+         * may get passed with `add`, `addDir`, and `change` events, set this to `true` to ensure it is
+         * provided even in cases where it wasn't already available from the underlying watch events.
+         */
+        alwaysStat?: boolean;
+
+        /**
+         * If set, limits how many levels of subdirectories will be traversed.
+         */
+        depth?: number;
+
+        /**
+         * Interval of file system polling.
+         */
+        interval?: number;
+
+        /**
+         * Interval of file system polling for binary files. ([see list of binary extensions](https://gi
+         * thub.com/sindresorhus/binary-extensions/blob/master/binary-extensions.json))
+         */
+        binaryInterval?: number;
+
+        /**
+         *  Indicates whether to watch files that don't have read permissions if possible. If watching
+         *  fails due to `EPERM` or `EACCES` with this set to `true`, the errors will be suppressed
+         *  silently.
+         */
+        ignorePermissionErrors?: boolean;
+
+        /**
+         * `true` if `useFsEvents` and `usePolling` are `false`). Automatically filters out artifacts
+         * that occur when using editors that use "atomic writes" instead of writing directly to the
+         * source file. If a file is re-added within 100 ms of being deleted, Chokidar emits a `change`
+         * event rather than `unlink` then `add`. If the default of 100 ms does not work well for you,
+         * you can override it by setting `atomic` to a custom value, in milliseconds.
+         */
+        atomic?: boolean | number;
+
+        /**
+         * can be set to an object in order to adjust timing params:
+         */
+        awaitWriteFinish?: AwaitWriteFinishOptions | boolean;
+    }
+
+    // copied from https://github.com/paulmillr/chokidar/blob/5e09c85ea04ffb7fd9cdb829892540e93ce63553/types/index.d.ts
+    interface AwaitWriteFinishOptions {
+        /**
+         * Amount of time in milliseconds for a file size to remain constant before emitting its event.
+         */
+        stabilityThreshold?: number;
+
+        /**
+         * File size polling interval.
+         */
+        pollInterval?: number;
+    }
+}
+
+export = GlobWatcher;
+export as namespace GlobWatcher;
```

Comment:
I believe this is incorrect. The package doesn't provide a umd module or uses the global scope.

Reply:
True, but I'd like to access the exported types, specifically line 58 of the test file: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34839/files#diff-2be02a26dd97ea05d84195b762a71911R58
How can I make `WatchOptions` accessible?

Reply:
You should be fine by removing this line and using `import =` syntax on the test file.